index.html

```diff
@@ -66,6 +66,157 @@ <h1>
+                <p>html</p>
+            </div>
+            <button type="button">
+                See Project
+            </button>
+        </article>
+        <article class="card-work">
+            <div class="card-image">
+                <img src="https://source.unsplash.com/327x474/?cat,library" alt="image work">
+            </div>
+            <h2 class="card-title">
+                Multi-Post Stories Gain+Glory
+            </h2>
+            <div class="card-tags">
+                <p>Rubi on rails</p>
+                <p>css</p>
+                <p>JavaScript</p>
+                <p>html</p>
+            </div>
+            <button type="button">
+                See Project
+            </button>
+        </article>
+        <article class="card-work">
+            <div class="card-image">
+                <img src="https://source.unsplash.com/327x474/?cat,library" alt="image work">
+            </div>
+            <h2 class="card-title">
+                Multi-Post Stories Gain+Glory
+            </h2>
+            <div class="card-tags">
+                <p>Rubi on rails</p>
+                <p>css</p>
+                <p>JavaScript</p>
+                <p>html</p>
+            </div>
+            <button type="button">
+                See Project
+            </button>
+        </article>
+        <article class="card-work">
+            <div class="card-image">
+                <img src="https://source.unsplash.com/327x474/?cat,library" alt="image work">
+            </div>
+            <h2 class="card-title">
+                Multi-Post Stories Gain+Glory
+            </h2>
+            <div class="card-tags">
+                <p>Rubi on rails</p>
+                <p>css</p>
+                <p>JavaScript</p>
+                <p>html</p>
+            </div>
+            <button type="button">
+                See Project
+            </button>
+        </article>
+    </section>
+    <section class="about-myself">
+        <h1>About me</h1>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded,
+            don’t hestiate to contact me.
+        </p>
+        <button type="button">
+            Get My Resume
+        </button>
+        <ul class="categorie-about-myself">
+            <li>
+                <article class="card-categories">
+                    <div class="categorie-image">
+                        <img src="./assest/icon-languages.svg" alt="icon languages">
+                    </div>
+                    <h2 class="categorie-title">
+                        Languages
+                    </h2>
+                    <div class="categorie-tags">
+                        <p>React</p>
+                        <p>Rubi on rails</p>
+                        <p>css</p>
+                        <p>JavaScript</p>
+                        <p>html</p>
+                    </div>
```

Comment:
- [ ] Consider using a `ul` tag to wrap the category technologies and each of the technology is a `li` tag.
This is because the category technologies is a list of unordered items and it would be semantically correct to wrap them in a `ul` tag.

```suggestion
                    <ul class="categorie-tags">
                        <li>React</li>
                        <li>Rubi on rails</li>
                        <li>css</li>
                        <li>JavaScript</li>
                        <li>html</li>
                    </ul>
```